gateway/src/main/java/discord4j/gateway/IdentifyOptions.java

```diff
@@ -41,6 +44,20 @@
     @Nullable
     private volatile String resumeSessionId;
 
+    /**
+     * Create a new identifying policy.
+     *
+     * @param shardInfo shard index and count the client using this policy will identify with
+     * @param initialStatus initial presence status the bot will identify with
+     * @param intents the raw intent value to subscribe to
+     */
+    public IdentifyOptions(ShardInfo shardInfo, @Nullable StatusUpdate initialStatus, long intents) {
+        this.shardInfo = shardInfo;
+        this.initialStatus = initialStatus;
+        this.intents = intents;
+        this.guildSubscriptions = false;
```

Comment:
Why is guildSubscription false by default ?

Reply:
When intents are set, guild subscription is false by default, because intents cover more than the `guildSubscription` field. In case the intents are set, all events which would be subscribed by an subscription could be subscribed via intent.

Reply:
Interesting, thanks!

Reply:
You're welcome :)